R/utils.R

```diff
@@ -41,9 +41,11 @@ load_model <- function(path) {
 #' @param ... Passed on to the `plot_vars()` or `plot_nodes()` function
 #'            in the `liesel.model.viz` module.
 #'
+#' @importFrom reticulate py_require
 #' @export
 
 plot.liesel.model.model.Model <- function(x, nodes = FALSE, ...) {
+  try(py_require("pygraphviz"), silent = TRUE)
```

Comment:
I guess we would have to update this to pydot if the corresponding PR in Liesel is merged?

Reply:
Exactly, but `py_require()` pulls the Python dependencies from PyPI, so we would need to make another release before the switch from PyGraphviz to pydot arrives in RLiesel. Anyway, this line is optional. We can also remove it now and leave the installation of either PyGraphviz or pydot up to the user.

Reply:
Makes sense. Since this line will probably be out of date very soon, and action from the user will most often be necessary to successfully install pygraphviz anyway, I think I'm in favor of already removing it now. Then we don't have to make another rliesel once the PR in liesel is merged.

Reply:
Agreed and done.